dotnet8/hello-pt/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,73 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Sample SAM Template with Powertools for AWS Lambda (.NET) for {{ cookiecutter.project_name }}
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 10
+    MemorySize: 256
```

Comment:
As mentioned above can we use 512 as the default for API Gateway based functions?